src/components/Home/singleBook/singleBook.css

```diff
@@ -1,10 +1,92 @@
 .sectionone__container__contents__content {
   margin-bottom: 15px;
-  border: 1px solid;
+  border: 1px solid #0290ff;
+  border-radius: 5px;
+  padding: 20px;
+  background-color: #fff;
 }
 
 .sectionone__container__contents__aside {
   display: flex;
   justify-content: space-between;
   align-items: center;
 }
+
+.sectionone__container__contents__aside__left > #title,
+.sectionone__container__contents__aside__left > #category {
```

Comment:
- [ ] Using Inefficient CSS Selectors is not good practice. [When you are using selectors, the longer it is, the higher specificity it will comprise. This can break the CSS cascade and mess up the styles of your site.](https://speckyboy.com/good-bad-css-practices/#:~:text=When%20you%20are%20using%20selectors%2C%20the%20longer%20it%20is%2C%20the%20higher%20specificity%20it%20will%20comprise.%20This%20can%20break%20the%20CSS%20cascade%20and%20mess%20up%20the%20styles%20of%20your%20site.). 

Kindly not use `ids` with long selectors 😊